pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -77,12 +77,15 @@ var (
 	//on devices without a TPM
 	quoteKeyFile = types.CertificateDirname + "/attest.key.pem"
 
-	// this PCRSelections is used as an entropy to generate keys and the selection
+	// PCRSelection is used as an entropy to generate keys and the selection
 	// of PCRs do not matter as well as the contents but PCR[7] is not changed often
 	// on our devices
-	pcrSelection     = tpm2.PCRSelection{Hash: tpm2.AlgSHA256, PCRs: []int{7}}
-	pcrListForQuote  = tpm2.PCRSelection{Hash: tpm2.AlgSHA256, PCRs: []int{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}}
-	defaultKeyParams = tpm2.Public{
+	PcrSelection = tpm2.PCRSelection{Hash: tpm2.AlgSHA256, PCRs: []int{7}}
```

Comment:
Do you plan to import this from somewhere? We usually don't import `../cmd/..` packages as they are intended to build the final "binaries". Would it make sense to move the exported variables and functions to `pillar/evetpm`?

Reply:
good point, yes!